lib/node_modules/@stdlib/lapack/base/dlacpy/src/addon.c

```diff
@@ -0,0 +1,83 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+#include "stdlib/lapack/base/dlacpy.h"
+#include "stdlib/blas/base/shared.h"
+#include "stdlib/napi/export.h"
+#include "stdlib/napi/argv.h"
+#include "stdlib/napi/argv_int32.h"
+#include "stdlib/napi/argv_int64.h"
+#include "stdlib/napi/argv_strided_float64array.h"
+#include <node_api.h>
+
+/**
+* Receives JavaScript callback invocation data.
+*
+* @param env    environment under which the function is invoked
+* @param info   callback data
+* @return       Node-API value
+*/
+static napi_value addon( napi_env env, napi_callback_info info ) {
+	STDLIB_NAPI_ARGV( env, info, argv, argc, 8 );
+
+	STDLIB_NAPI_ARGV_INT32( env, order, argv, 0 );
+	STDLIB_NAPI_ARGV_INT32( env, uplo, argv, 1 );
+
+	STDLIB_NAPI_ARGV_INT64( env, M, argv, 2 );
+	STDLIB_NAPI_ARGV_INT64( env, N, argv, 3 );
+	STDLIB_NAPI_ARGV_INT64( env, LDA, argv, 5 );
+	STDLIB_NAPI_ARGV_INT64( env, LDB, argv, 7 );
+
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT64ARRAY( env, A, M*N, 1, argv, 4 );
```

Comment:
This is wrong. You need to be using `argv_strided_float64array2d`. Applies here and below.